CI-CD/Bamboo/ws_scan.sh

```diff
@@ -5,8 +5,13 @@
 # WS_WSS_URL = https://saas.whitesourcesoftware.com/agent
```

Comment:
@tabac-ws If you're adding the env vars as part of the script itself, you can remove them from the comments, no?

Reply:
Hi @tidharm ,
Sadly, no. 
Since as best practice, the apikey and userKey should be masked you can't adding them as part of the script. 
When adding them as env vars of Bamboo, it added a prefix of "bamboo_" which is not picked up by the UA. 
This is the reason to add in the script the proper WS_ prefix variables which are pointed to Bamboo env vars.

Reply:
@tabac-ws I think I didn't explain myself properly, let's PM